asv.conf.json

```diff
@@ -79,10 +80,10 @@
     //     Platform, as in sys.platform. Possible values for the common
     //     cases: 'linux2', 'win32', 'cygwin', 'darwin'.
     //
-    // "exclude": [
+    "exclude": [
     //     {"python": "3.2", "sys_platform": "win32"}, // skip py3.2 on windows
-    //     {"environment_type": "conda", "six": null}, // don't run without six on conda
-    // ],
+         {"environment_type": "conda", "six": null}, // don't run without six on conda
```

Comment:
Could you elaborate on the story behind this?

Reply:
@soupault Ah yes, I was there and quite puzzled by this! Basically, we don't need six anymore... But airspeed velocity does!!! 🤦‍♂️